test/parallel/test-worker-nexttick-terminate.js

```diff
@@ -13,13 +13,6 @@ process.nextTick(() => {
 `, { eval: true });
 
 // Test deprecation of .terminate() with callback.
```

Comment:
I don't know if we should keep the test without callback, but at least delete the comment.